router/functionHandler.go

```diff
@@ -93,6 +95,12 @@ func (fh *functionHandler) tapService(serviceUrl *url.URL) {
 func (fh *functionHandler) handler(responseWriter http.ResponseWriter, request *http.Request) {
 	reqStartTime := time.Now()
 
+	// retrieve url params and add them to request header
+	vars := mux.Vars(request)
+	for k, v := range vars {
+		request.Header.Add(fmt.Sprintf("X-Fission-Params-%v", k), v)
```

Comment:
What happens if there's a url like "http://example.com/foo: bar/"?  Both ":" and " " are valid chars in URLs.